hw2/05-form.js

```diff
@@ -1 +1,60 @@
-// Add your code here
+//Add your code here
+document.addEventListener('DOMContentLoaded', function() {
+    const form = document.querySelector('form');
+
+    form.addEventListener('submit', function(event) {
+        
+        event.preventDefault(); 
+
+        // Reading User Inputs
+        const fullName = document.getElementById('fullname').value;
+        const email = document.getElementById('exampleInputEmail1').value;
+        const registrationStatus = document.querySelector('.form-select').value;
+        const courses = document.querySelectorAll('.form-check-input:checked');
+        const coursesTaken = Array.from(courses).map(course => course.nextElementSibling.textContent);
+
+        // modal
+        const message = `
+            <p><strong>Full Name:</strong> ${fullName}</p>
+            <p><strong>Email:</strong> ${email}</p>
+            <p><strong>Registration Status:</strong> ${registrationStatus}</p>
+            <p><strong>Courses Taken:</strong> ${coursesTaken.join(', ')}</p>
+        `;
+
+        // Create a modal and display the message
+        const modal = document.createElement('div');
+        modal.classList.add('modal');
+        modal.innerHTML = `
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 class="modal-title">User Input</h5>
+                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    </div>
+                    <div class="modal-body">
+                        ${message}
+                    </div>
+                    <div class="modal-footer">
+                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                    </div>
+                </div>
+            </div>
+        `;
+
+        // Append the modal to the document body
+        document.body.appendChild(modal);
```

Comment:
Did the assignment ask the information to be displayed in the html doc, or just the console/Log. Either way, it is nice to view the result on the html after being submitted.